CMakePresets.json

```diff
@@ -0,0 +1,234 @@
+{
+	"version": 7,
+	"cmakeMinimumRequired": {
+		"major": 3,
+		"minor": 27,
+		"patch": 0
+	},
+	"configurePresets": [
+		{
+			"name": "vcpkg",
+			"displayName": "Configure with vcpkg. please set VCPKG_ROOT environment first",
+			"toolchainFile": "$env{VCPKG_ROOT}/scripts/buildsystems/vcpkg.cmake"
+		},
+
+		{
+			"name": "default",
+			"displayName": "Default freerdp library",
+			"description": "Default freerdp library with vcpkg",
+			"inherits": "vcpkg",
+			"cacheVariables": {
+				"WITH_FFMPEG": "OFF",
+				"WITH_SWSCALE": "OFF",
+				"CHANNEL_URBDRC": "OFF",
+				"WITH_PKCS11": "OFF",
+				"WITH_CUPS": "OFF",
+				"WITH_MANPAGES": "OFF",
+				"WITH_KRB5": "OFF",
+				"WITH_CLIENT": "OFF",
+				"WITH_SERVER": "OFF",
+				"WITH_PROXY": "OFF"
```

Comment:
`WITH_SERVER=OFF` also does not build the `libfreerdp-server3.so` (we did a better dependency with `WITH_CLIENT=OFF` where the library is outside with `WITH_CLIENT_COMMON`)
so better keep that on and deactivate the components `WITH_SHADOW`, `WITH_PROXY` and `WITH_PLATFORM_SERVER`